css/mainStyle.css

```diff
@@ -56,7 +56,27 @@ h2 {
     align-content: left;
     font-weight: 500; 
 }
+
+/* calibrate button */
+.roundButton {
```

Comment:
Instead of having a class for the calibrate button, could a style be applied to all buttons and have id assigned to the calibrate button like `#calibrate`?